src/main/kotlin/de/livepoll/api/service/PollItemService.kt

```diff
@@ -67,6 +65,12 @@ class PollItemService {
 
     //-------------------------------------------- Create --------------------------------------------------------------
 
+//    fun ensurePositionNotTaken(pollItems: MutableList<PollItem>, itemPos: Int) {
+//        if (pollItems.map { it.position }.contains(itemPos)) {
+//            throw ResponseStatusException(HttpStatus.BAD_REQUEST, "Position already taken by another poll item")
+//        }
+//    }
+
```

Comment:
Can we delete this function or do you need it in the future?

Reply:
You're right, we don't need this anymore, this was before the new logic where new poll items get appended to the end automatically. I will delete it.